Resources/Prototypes/_NF/Entities/Clothing/OuterClothing/hardsuits.yml

```
@@ -175,7 +175,8 @@
 - type: entity
   parent: ClothingOuterHardsuitNfsdBronze
   id: ClothingOuterHardsuitNfsdSilver
-  name: NFSD silver patrol hardsuit
+  name: NFSD silver patrol suit
+  description: A lightly armored suit that protects from the danger of space, standard issue to be worn by NFSD on patrol. This silver variant has standard rank markings.
```

```suggestion
  description: The standard-issue NFSD space proof suit for patrol duty. This silver variant has officer rank markings.
```
Shorter.  While not _technically_ correct, I think saying officer for the grouping of "deputies and detective" is fine.  Others may disagree.